ngraph/frontend/onnx_import/src/ops_bridge.cpp

```diff
@@ -194,9 +196,49 @@ namespace ngraph
             }
         }
 
+        void OperatorsBridge::_unregister_operator(const std::string& name,
+                                                   std::int64_t version,
+                                                   const std::string& domain)
+        {
+            std::lock_guard<std::mutex> guard(lock);
+
+            auto domain_it = m_map.find(domain);
+            if (domain_it == m_map.end())
+            {
+                NGRAPH_ERR << "unregister_operator: domain '" + domain +
+                                  "' was not registered before";
+                return;
+            }
+            auto name_it = domain_it->second.find(name);
+            if (name_it == domain_it->second.end())
+            {
+                NGRAPH_ERR << "unregister_operator: operator '" + name +
+                                  "' was not registered before";
+                return;
+            }
+            auto version_it = name_it->second.find(version);
+            if (version_it == name_it->second.end())
+            {
+                NGRAPH_ERR << "unregister_operator: operator '" + name + "' with version " +
+                                  std::to_string(version) + " was not registered before";
+                return;
+            }
+            m_map[domain][name].erase(version_it);
+            if (m_map[domain][name].empty())
+            {
+                m_map[domain].erase(name);
+                if (m_map[domain].empty())
+                {
+                    m_map.erase(domain);
+                }
+            }
+        }
```

Comment:
Since the `find` method is overwrited in the line 159 I wonder if it works strict enough.
example:
Add in opset 99 has been registered in the default domain by the user.
User is trying to unregister Add from opset 99 twice.
Will it throw an error or unregister custom Add opset 99 as well as the latest "Default" Add?
Could you verify it?

Reply:
This case works. I'm not using details::find function in unregister_operator